lib/bacalhau.js

```diff
@@ -21,7 +21,8 @@ export async function install () {
       { platform: 'darwin', arch: 'arm64', asset: `bacalhau_${DIST_TAG}_darwin_arm64.tar.gz` },
       { platform: 'linux', arch: 'x64', asset: `bacalhau_${DIST_TAG}_linux_amd64.tar.gz` },
       { platform: 'linux', arch: 'arm64', asset: `bacalhau_${DIST_TAG}_linux_arm64.tar.gz` },
-      { platform: 'win32', arch: 'x64', asset: `bacalhau_${DIST_TAG}_windows_amd64.tar.gz` }
+      { platform: 'win32', arch: 'x64', asset: `bacalhau_${DIST_TAG}_windows_amd64.tar.gz` },
+      { platform: 'linux', arch: 'arm', asset: `bacalhau_${DIST_TAG}_linux_arm6.tar.gz` }
```

Comment:
Would you mind keeping this list sorted by platform? (Add the new line before the win32 entry, not after.)

```suggestion
      { platform: 'linux', arch: 'arm', asset: `bacalhau_${DIST_TAG}_linux_arm6.tar.gz` },
      { platform: 'win32', arch: 'x64', asset: `bacalhau_${DIST_TAG}_windows_amd64.tar.gz` }
```